packages/combo-box/src/vaadin-combo-box-light.js

```diff
@@ -118,10 +118,14 @@ class ComboBoxLight extends ComboBoxDataProviderMixin(ComboBoxMixin(ValidateMixi
   }
 
   /**
-   * @return {string}
+   * This method from InputMixin is overridden to make
+   * the input element value property customizable.
```

Comment:
Note: strictly speaking, this is a getter and not a method. Let's put it this way

```suggestion
   * Override getter from `InputMixin` to allow using arbitrary property 
   * name instead of `value` for accessing the input element value.
```

Reply:
Done.